types/livereload/index.d.ts

```diff
@@ -44,7 +44,7 @@ interface CreateServerConfig extends ServerConfig {
 }
 
 /** Live Reload Server object, provides main functionality */
-declare class LiveReloadServer extends EventEmitter {
+export class LiveReloadServer extends EventEmitter {
     config: ServerConfig;
```

Comment:
Seems like you might as well export `ServerConfig` etc as well.

Reply:
The package is not mine but you are right about this. I will fix that.

Reply:
Fixed